commerce_coordinator/apps/commercetools/catalog_info/edx_utils.py

```diff
@@ -65,6 +65,12 @@ def get_edx_payment_intent_id(order: CTOrder) -> Union[str, None]:
     return None
 
 
+def get_edx_payment_service_provider(order: CTOrder) -> Union[str, None]:
+    for pr in order.payment_info.payments:
+        return pr.obj.payment_method_info.payment_interface
```

Comment:
This loop is kind of same as doing 
```
if len(order.payment_info.payments) > 0:
    order.payment_info.payments[0].obj.payment_method_info.payment_interface
```
Why are we looping over it if it returns the info related to the first obj?